.github/workflows/ci.yml

```diff
@@ -120,4 +120,21 @@ jobs:
             make || exit 1; \
             cd ..; \
           done;
-
+  cmake-windows-latest:
+    runs-on: windows-latest
+    env:
+      APACHE_LOUNGE_DISTRO_VERSION: 2.4.57
+      WORKSPACE: 'D:\a\mod_proxy_cluster\mod_proxy_cluster\'
+      HTTPD_DEV_HOME: 'D:\a\mod_proxy_cluster\mod_proxy_cluster\httpd-apache-lounge\Apache24'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup Developer Command Prompt for Microsoft Visual C++
+        uses: ilammy/msvc-dev-cmd@v1
+      - name: Get httpd
+        run: |
+          curl.exe --output "httpd-2.4.57-win64-VS16.zip" --url "https://www.apachelounge.com/download/VS16/binaries/httpd-2.4.57-win64-VS16.zip"
+          Expand-Archive -Path "httpd-2.4.57-win64-VS16.zip" -DestinationPath "httpd-apache-lounge"
+      - name: Build
+        run: |
+          ./.github/workflows/windows-build.bat
```

Comment:
Am I wrong to assume this build file is usable outside of GitHub Actions? I think this should be in `/native` rather than in `.github/`.

Reply:
Done, it should be a bit cleaner now.

Reply:
Perfect